toolchain/lexer/tokenized_buffer_test.cpp

```diff
@@ -31,19 +31,24 @@ using ::testing::StrEq;
 
 class LexerTest : public ::testing::Test {
  protected:
-  auto GetSourceBuffer(llvm::Twine text) -> SourceBuffer& {
-    source_storage.push_back(
-        std::move(*SourceBuffer::CreateFromText(text.str())));
-    return source_storage.back();
+  auto GetSourceBuffer(llvm::StringRef text) -> SourceBuffer& {
+    std::string filename = llvm::formatv("test{0}.carbon", ++file_index);
+    CARBON_CHECK(fs.addFile(filename, /*ModificationTime=*/0,
+                            llvm::MemoryBuffer::getMemBuffer(text)));
+    source_storage.push_front(
+        std::move(*SourceBuffer::CreateFromFile(fs, filename)));
+    return source_storage.front();
   }
 
-  auto Lex(llvm::Twine text,
+  auto Lex(llvm::StringRef text,
            DiagnosticConsumer& consumer = ConsoleDiagnosticConsumer())
       -> TokenizedBuffer {
     return TokenizedBuffer::Lex(GetSourceBuffer(text), consumer);
   }
 
-  llvm::SmallVector<SourceBuffer, 16> source_storage;
+  llvm::vfs::InMemoryFileSystem fs;
+  int file_index = 0;
+  std::forward_list<SourceBuffer> source_storage;
```

Comment:
Shouldn't these have a `_` at the end of their names?

Reply:
Done -- I'd been leaving source_storage alone initially and aiming for consistency, but now I suppose I'm making up the majority.